mission/functions/core/ui/taskroster/fn_tr_supportTask_map_hide.sqf

```diff
@@ -17,10 +17,5 @@
 disableSerialization;
 #include "..\..\..\..\config\ui\ui_def_base.inc"
 
-private _display = VN_DISP_TR_TASKROSTER;
-
-{
-	ctrlDelete _x;
-}forEach [VN_TR_SUPREQ_MAP_CTRL, VN_TR_SUPREQ_ACCEPT_CTRL, VN_TR_SUPREQ_ABORT_CTRL];
-
-VN_TR_SUPREQ_SELPOS_CTRL ctrlenable true;
\ No newline at end of file
+_markerName = format["%1_missionMarker", getPlayerUID player];
+deleteMarkerLocal _markerName;
```

Comment:
we only need to delete any existing map markers now because we aren't maki9ng everything super dynamic and overly complicated.